app/models/org.rb

```diff
@@ -0,0 +1,20 @@
+class Org < ApplicationRecord
```

Comment:
I remember the organisation vs organization argument for using `Org`, but it's hard to move past this feeling that using `Org` is a bit lazy here, where everything else in the repo has their full labels.

How about we just pull the trigger on `Organization`. Thoughts @indirect @martinemde ?

Reply:
We could change it if you think it's ambiguous, but it's more than just spelling. Typing less thousands of times over the life of the model is worth it. There's plenty of examples in this repo and related ones where we used a short word when a longer one was available.

I suggest we use the shortest unambiguous name in the code.

...or there's always the slightly longer "Orga" (my daughter's word for "Orchid") 😆

Reply:
I would prefer `Organization` over `Org` because if we keep the term consistent to `Organization`, then there will be one consistent naming of the term in the repo. There won't be any ambiguity between referring to the concept as organisation / organization / org.

Reply:
👍🏻 to organization from me

Reply:
@indirect saw this has been marked as resolved, but the code hasn't changed in regards to this. Did you end up making a decision?

Reply:
@colby-swandale whoops, sorry about that. I don't personally think the long version is worth it, but I'm fine with the majority team consensus to go with `Organization`. I just pushed a commit that renames everything, right before I saw your comment. 😅